include/nuttx/note/note_driver.h

```diff
@@ -35,6 +39,72 @@
  * Public Types
  ****************************************************************************/
 
+struct note_driver_s;
+
+struct note_driver_ops_s
+{
+  CODE void (*write)(FAR struct note_driver_s *drv, FAR const void *note,
+                     size_t notelen);
+  CODE void (*start)(FAR struct note_driver_s *drv, FAR struct tcb_s *tcb);
+  CODE void (*stop)(FAR struct note_driver_s *drv, FAR struct tcb_s *tcb);
+#ifdef CONFIG_SCHED_INSTRUMENTATION_SWITCH
+  CODE void (*suspend)(FAR struct note_driver_s *drv, FAR struct tcb_s *tcb);
+  CODE void (*resume)(FAR struct note_driver_s *drv, FAR struct tcb_s *tcb);
+#ifdef CONFIG_SMP
```

Comment:
```suggestion
#  ifdef CONFIG_SMP
```

Reply:
The style is now consistent, this change is not required

Reply:
done